playbooks/modules/s3_key.yml

```diff
@@ -0,0 +1,73 @@
+---
+- name: S3 Key Module Operations on PowerScale Storage
+  hosts: localhost
+  connection: local
+  vars:
+    onefs_host: "**.**.**.**"
+    port_no: "0000"
+    api_user: "user"
+    api_password: "pass"
+    verify_ssl: false
+    access_zone: "sample-zone"
+    user: "sample-user"
+
+  tasks:
+    - name: Create S3 Key - Check_mode
+      dellemc.powerscale.s3_key:
+        onefs_host: "{{onefs_host}}"
+        port_no: "{{port_no}}"
```

Comment:
@fpfuetsch, thank you for making all the required changes. Changes look good to me. If I may ask for one more favor, would you be okay with fixing these minor warnings around Jinja2 spacing?